test/extended/cluster/cl.go

```diff
@@ -56,77 +59,18 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 	})
 
 	g.It("should load the cluster", func() {
-		project := ConfigContext.ClusterLoader.Projects
-		tuningSets := ConfigContext.ClusterLoader.TuningSets
-		sync := ConfigContext.ClusterLoader.Sync
-		if project == nil {
-			e2e.Failf("Invalid config file.\nFile: %v", project)
+		projects := ConfigContext.ClusterLoader.Projects
+		if projects == nil {
+			e2e.Failf("Invalid config file.\nFile: %v", projects)
 		}
-
-		var namespaces []string
-		//totalPods := 0 // Keep track of how many pods for stepping
-		// TODO sjug: add concurrency
+		sync := ConfigContext.ClusterLoader.Sync
```

Comment:
is this intending to initialize something? or just shorten the variable name used? Cause `sync` var name is now shadowing the imported package `sync` ...

Reply:
Just to shorten the variable name